src/lib/routes/unleash-types.ts

```diff
@@ -8,7 +8,7 @@ export interface IAuthRequest<
     ReqQuery = any,
 > extends Request<PARAM, ResBody, ReqBody, ReqQuery> {
     user: User;
-    logout: () => void;
+    logout: (() => void) | ((callback: (err: any) => void) => void);
```

Comment:
```suggestion
    logout: (() => void) | ((callback: (err?: any) => void) => void);
```
Maybe this is a bit more explicit as when there's success passport does not send the err: https://github.com/jaredhanson/passport/blob/1e8f112bd233dbffb1904d4dd2051780d81b0a22/lib/sessionmanager.js#L90 and also aligns well with the test: https://github.com/Unleash/unleash/pull/2041/files#diff-fef84c04ae9c0e2de267923619bf716cc198f58a0f5fd45cef15ff8f1ef34e99R121